crates/telio-lana/src/lib.rs

```diff
@@ -115,3 +135,196 @@ pub fn deinit_lana() -> Result<moose::Result, moose::Error> {
 pub fn is_lana_initialized() -> bool {
     MOOSE_INITIALIZED.load(DEFAULT_ORDERING)
```

Comment:
One thing I missed when we talked is this global flag. It requires us to do 2 things:
- restore it to false at the end of each test
- since we can't/won't change it to thread local, we need to mark all tests in this file to run sequentially (for example with `serial_test` crate)
The second point means that we might not need to put mock in thread local storage and a single global mutex protected variable might be enough.

Reply:
- Restored to false at the end of each test on teardown().
- Tests now run sequentially.